app/upgrades/v11/upgrades.go

```diff
@@ -1,9 +1,11 @@
 package v11
 
 import (
+	"context"
```

Comment:
I think we can afford to delete all old upgrade code.

It cannot be executed correctly even if you change the imports and wiring, so we might as well delete them now and leave only `app/upgrades/v19`.